### ISSUE TITLE
Creating a configuration breaks in Python 2

### ISSUE THREAD

**Issue by GlennS** [linode/python-linode-api#32]
I am unable to create a config on a Linode instance. When I try to call create_config, I get an ImportError for linode.mappings. This breaks in Python 2, but works fine in Python 3.

Using Python 2.7.13 and linode-api 4.1.1b1 (which looks like it should include the possibly related commit https://github.com/linode/python-linode-api/commit/57e70ee9523789ccf4d0e1080c217422febce764 ).

This is probably something to do with the different import behaviour in Python 2 and Python 3.

First install the library and launch python:
```
virtualenv -p python2 venv
source venv/bin/activate
pip install --pre linode-api
python
```

Now run the following (filling in your personal API key):
```
import linode

personal_api_key = "<fill your key in here>"
plan_id = "g5-nanode-1"
region_id = "eu-west-1a"
region = None

client = linode.LinodeClient(personal_api_key)

for r in client.get_regions():
    if region_id == r.id:
        region = r

if region is None:
    raise ValueError("Unknown region id %s" % region_id)

plan = None
for p in client.linode.get_types():
    if plan_id == p.id:
        plan = p

if plan is None:
    raise ValueError("Unknown plan id %s" % plan_id)

service = linode.Service(client, plan_id)

instance = client.linode.create_instance(service, region)

# TODO: wait for instance creation

main_disk = instance.create_disk(
    size = plan.storage,
    label = "OS",
    filesystem = "ext4",
    read_only = False
)

kernel_direct_disk = None
for k in client.linode.get_kernels():
    if k.deprecated or not k.x64 or not k.kvm:
        pass
    elif k.label == "Direct Disk":
        kernel_direct_disk = k


if kernel_direct_disk is None:
    raise ValueError("Unknown kernel id %s" % kernel_id)

## This call causes the problem
config = instance.create_config(
    kernel = kernel_direct_disk,
    label = "my-machine",
    disks = [main_disk]
)
```

The stack trace for the error looks like this:
```
Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
  File "/home/glenn/linode-api-test/venv/local/lib/python2.7/site-packages/linode/objects/linode/linode.py", line 213, in create_config
    c._populate(result)
  File "/home/glenn/linode-api-test/venv/local/lib/python2.7/site-packages/linode/objects/linode/config.py", line 36, in _populate
    import linode.mappings as mapper
ImportError: No module named mappings
```

**Comment by Dorthu** [comment]
Thanks for reporting, I'll take a look tonight.

**Comment by Dorthu** [comment]
This is fixed in version 4.1.1b2

**Comment by GlennS** [comment]
Great job, thanks.
